.ci/pytorch/test.sh

```diff
@@ -105,14 +105,6 @@ if [[ "$TEST_CONFIG" == *crossref* ]]; then
   export PYTORCH_TEST_WITH_CROSSREF=1
 fi
 
-if [[ "$TEST_CONFIG" == *dynamo* ]]; then
-  export PYTORCH_TEST_WITH_DYNAMO=1
-fi
-
-if [[ "$TEST_CONFIG" == *inductor* ]]; then
-  export PYTORCH_TEST_WITH_INDUCTOR=1
-fi
-
```

Comment:
This feels wrong without removing from https://github.com/pytorch/pytorch/blob/87b9ab48704bcd86753afd7166ef8b01011ad3fa/.github/workflows/pull.yml#L127

(Because if you remove it like that, CI will still run dynamo/inductor shards but will be running some unrelated tests there.)